conda_smithy/templates/run_osx_build.sh.tmpl

```diff
@@ -9,6 +9,15 @@ set -xe
 MINIFORGE_HOME=${MINIFORGE_HOME:-${HOME}/miniforge3}
 MINIFORGE_HOME=${MINIFORGE_HOME%/} # remove trailing slash
```

Comment:
I think we can include [this blob](https://github.com/conda-forge/staged-recipes/blob/778e1c767ae07a9ba9eecc60e927325ce05f93d3/.scripts/run_osx_build.sh#L9-L11) from `staged-recipes` here instead.